front/src/components/login.js

```diff
@@ -18,21 +20,30 @@ export default class Login extends Component {
     }
 
     login(event) {
-        alert(this.state.email + "\n"
-            + this.state.password
-        );
-
         event.preventDefault();
+        AuthenticationService
+            .executeAuthenticationService(this.state.email, this.state.password)
+            .then(() => {
+                AuthenticationService.registerSuccessfulLogin(this.state.email, this.state.password);
+                this.props.history.push("/");
+            })
+            .catch(() => {
+                this.setState({error: true});
+                alert("Something went wrong")
+            });
+
     }
 
     render() {
+        if (AuthenticationService.isUserLoggedIn()) {
+            return <Redirect to={"/"}/>
+        }
         return (
             <form onSubmit={this.login}>
                 <h3>Sign In</h3>
-
                 <div className="form-group">
                     <label>Email address</label>
-                    <input name="email" onChange={this.handleChange} type="email" className="form-control"
```

Comment:
why is it removed? with tgis validation of email was performed (e.g. missing `@` sign)